adafruit_epd/epd.py

```diff
@@ -71,6 +71,11 @@ def __init__(self, width, height, spi, cs_pin, dc_pin, sramcs_pin, rst_pin, busy
 
         # SPI interface (required)
         self.spi_device = spi
+        while not self.spi_device.try_lock():
+            pass
+        self.spi_device.configure(baudrate=1000000) # 1 Mhz
```

Comment:
On the small chance you don’t know about the feature (and in spite of the fact that this is already merged), I’ll mention that 1_000_000 is easier to read than 1000000.